extensions/amp-base-carousel/1.0/README.md

```diff
@@ -0,0 +1,731 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content along a horizontal axis or vertical axis.
+---
+
+<!---
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Bento Carousel
+
+## Usage
+
+A generic carousel for displaying multiple similar pieces of content along a
+horizontal or vertical axis.
```

Comment:
I think it may be clarifying to state that this component can be used in multiple environments upfront. Maybe:

```suggestion
A generic carousel for displaying multiple similar pieces of content along a horizontal or vertical axis. Use Bento Carousel as a web component `<bento-base-carousel>`, or a Preact/React functional component. `<BentoBaseCarousel>`.
```